contracts/VaultV3.vy

```diff
@@ -1385,32 +1381,55 @@ def set_use_default_queue(use_default_queue: bool):
     log UpdateUseDefaultQueue(use_default_queue)
 
 @external
-def set_deposit_limit(deposit_limit: uint256):
+def set_deposit_limit(deposit_limit: uint256, override: bool = False):
     """
     @notice Set the new deposit limit.
     @dev Can not be changed if a deposit_limit_module
-    is set or if shutdown.
+    is set unless the override flag is true or if shutdown.
     @param deposit_limit The new deposit limit.
+    @param override If a `deposit_limit_module` already set should be overridden.
     """
     assert self.shutdown == False # Dev: shutdown
     self._enforce_role(msg.sender, Roles.DEPOSIT_LIMIT_MANAGER)
-    assert self.deposit_limit_module == empty(address), "using module"
+
+    # If we are overriding the deposit limit module.
+    if override:
+        # Make sure it is set to address 0 if not already.
+        if self.deposit_limit_module != empty(address):
+
+            self.deposit_limit_module = empty(address)
+            log UpdateDepositLimitModule(empty(address))
+    else:  
+        # Make sure the deposit_limit has been set to uint max.
```

Comment:
this check should always be equivalent to do doing this right? maybe do that one explicitly so that the comment make more sense?
`assert self.deposit_limit == max_value(uint256)`

Reply:
oh looks like i copied the code from the module and didnt update the comments